classes/output/report_table.php

```diff
@@ -126,7 +126,8 @@ public function __construct(string $uniqueid, string $baseurl, int $page = 0,
      * @return string html used to display the filename field.
      */
     public function col_filename($row) {
-        return \html_writer::link(new \moodle_url('/local/smartmedia/report_details.php', ['hash' => $row->contenthash]), $row->filename);
+        return \html_writer::link(new \moodle_url('/local/smartmedia/report_details.php',
+                ['hash' => $row->contenthash]), $row->filename);
```

Comment:
```suggestion
        $url = new \moodle_url('/local/smartmedia/report_details.php', ['hash' => $row->contenthash]);
        return \html_writer::link($url, $row->filename);
```

Reply:
If it's going to be on 2 lines anyways, might as well split it up and make it clearer I think. What do you think?